pkg/network/ebpf/c/http.h

```diff
@@ -167,51 +142,98 @@ static __always_inline void http_parse_data(char *p, http_packet_t *packet_type,
     }
 }
 
-static __always_inline int http_process(char *buffer, skb_info_t *skb_info, u16 src_port, u64 tags) {
+static __always_inline bool http_already_seen(http_transaction_t *http, skb_info_t *skb_info) {
+    if (!skb_info) {
+        return false;
+    }
+
+    if (http->tcp_seq == skb_info->tcp_seq) {
+        return true;
+    }
+
+    http->tcp_seq = skb_info->tcp_seq;
```

Comment:
This feels a bit strange because the function has a side effect. Calling this function twice with the same arguments would then have different results. Could you move the assignment outside this function?

Reply:
Sure!

Reply:
Done